src/graph/network.h

```diff
@@ -24,38 +25,66 @@ namespace network {
 // TODO(chao): Make this number configurable
 const int64_t kQueueSize = 200 * 1024 * 1024;
 
+/*!
+ * \brief Create NDArray from raw data
+ */
+NDArray CreateNDArrayFromRaw(std::vector<int64_t> shape,
+                             DLDataType dtype,
+                             DLContext ctx,
+                             void* raw);
+
 /*!
  * \brief Message type for DGL distributed training
  */
 enum MessageType {
- /*!
-  * \brief Message for send/recv NodeFlow
-  */
+  /*!
+   * \brief Message for send/recv NodeFlow
+   */
   kNodeFlowMsg = 0,
- /*!
-  * \brief Message for end-signal
-  */
-  kEndMsg = 1
+  /*!
+   * \brief Message for end-signal
+   */
+  kEndMsg = 1,
```

Comment:
please make the name consistent with the one in Python. In Python, it's called Final